### PR TITLE
Small change for ShortPixel mention

### PR DESCRIPTION
**Pull request by short-pixel-optimizer**
Small change for ShortPixel mention

### CODE REVIEW

File: src/content/en/fundamentals/performance/optimizing-content-efficiency/automating-image-optimization/index.md [hunk 1]
```diff
@@ -1410,9 +1410,8 @@ Cache Enabler requires the use of a sister program called Optimizer, which has
 an annual fee. This seems out of character for a genuinely open source solution.
 
 
-**Short Pixel** — Another option for use with Cache Enabler, also at a cost, is
-Short Pixel. Short Pixel functions much like Optimizer, described above. You can
-optimize up to 100 images a month for free.
+**ShortPixel** — Another option that can be used stand-alone or with Cache Enabler, also at a cost, is
+ShortPixel. In its stand alone use [ShortPixel](https://shortpixel.com) can add a &lt;picture&gt; tag that usually will serve the right type of image depending on the browser. You can optimize up to 100 images a month for free.
```
Comment (petele) [inline src/content/en/fundamentals/performance/optimizing-content-efficiency/automating-image-optimization/index.md]: * Please wrap lines at 80 characters
* Wrap "&amp;lt;picture&amp;gt;" in back ticks \` to mark as code, eg:
```
`&lt;picture&gt;`
```

**Comment by petele** [comment]
Also, please be sure to sign the CLA

**Comment by short-pixel-optimizer** [comment]
I signed it!

**Comment by petele** [comment]
@addyosmani - can you confirm you're happy with these changes.

**Comment by addyosmani** [comment]
@petele Look good to land. Over to you :)

**Pull request merged**